Source/WebCore/html/HTMLTextFormControlElement.cpp

```diff
@@ -155,6 +155,7 @@ void HTMLTextFormControlElement::didEditInnerTextValue(bool wasUserEdit)
     LOG(Editing, "HTMLTextFormControlElement %p didEditInnerTextValue", this);
 
     m_lastChangeWasUserEdit = wasUserEdit;
+    m_wasChangeEverUserEdit |= wasUserEdit;
```

Comment:
```suggestion
    m_wasEverChangedByUserEdit |= wasUserEdit;
```

Maybe `m_wasEverChangedByUserEdit` or `m_wasEverUserEdited` might read a bit more nicely?